app/adapters/cats.js

```diff
@@ -35,7 +35,8 @@ CATS.Adapter.Cats = Classify({
                         run.user = row.team_id;
                         run.contest = contest_id;
                         run.status = self.statuses[row['state']];
-                        run.start_processing_time = row['time'].to_date();
+                        var _time = CATS.App.utils.add_time(row['time'].to_date(), -times[run.user]);
```

Comment:
Why not `run.user.times_offset` here?
Also note that time-offset is described as 'seconds' in `model.js`. Either change comment there or scale the data accordingly.
Variable `_time` is not needed.